lib/saxerator/builder/xml_builder.rb

```diff
@@ -1,5 +1,11 @@
+require 'rexml/document'
+
 module Saxerator
   module Builder
+    class XmlDocument < REXML::Document
```

Comment:
@soulcutter Before this we had a Nokogiri document and he has a :to_xml method.
I thought that it'll be good to backward compatibility.
If it's bad Idea I can remove it